srcpkgs/go/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'go'
 pkgname=go
-version=1.11.5
-revision=2
+version=1.12
+revision=3
```

Comment:
It should be revision=1.

Reply:
Got it. To clarify my understand of the `template` file. Is the `revision` used to denote changes to the template when the `version` stays the same?

Reply:
Yes, that's right.

I can also be used to force a package rebuild when there is a change in the build infrastructure, e.g. compiler upgrade. It will likely happen to packages using Go compiler after this gets merged.